cg/models/workflow/mutant.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel
+import datetime as dt
+
+
+class MutantSampleConfig(BaseModel):
```

Comment:
We also need the following fields:
region_code e.g. "01"
lab_code, e.g. "SE100"
case_ID e.g. "merrymink"

Reply:
Currently used fields in mutant are:
region_code, lab_code, selection_criteria, case_ID, CG_ID_project, CG_ID_sample, Customer_ID_project, Customer_ID_sample.

Reply:
can we also have "priority" as field as well?

Reply:
done